cycledash/__init__.py

```diff
@@ -30,6 +31,51 @@ def internal_error(exception):
 def _configure_application(app):
     app.config.from_object('config')
 
+def _configure_templates(app):
+    # taken from http://stackoverflow.com/questions/1551382/user-friendly-time-format-in-python
+    @app.template_filter('humanize_date')
```

Comment:
I'm generally inclined not to like copy pasting library-ish code. What about https://github.com/jmoiron/humanize or something similar?